photon-lib/src/main/scala/com/linkedin/photon/ml/data/scoring/SimpleDataScores.scala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linkedin.photon.ml.data.scoring
+
+import org.apache.spark.rdd.RDD
+import org.apache.spark.rdd.RDD.rddToPairRDDFunctions
+
+/**
+ * The class used to track scored data points throughout training. The score objects are scores only, with no additional
+ * information.
+ *
+ * @param scores The scores consist of (unique ID, score) pairs as explained above.
+ */
+protected[ml] class SimpleDataScores(val scores: RDD[(Long, Double)])
```

Comment:
`CoordinateDataScores` here and `ModelDataScores` for the other one? It's a more descriptive and useful, because it orientates right away, and also explains right away why we have two.